ddtrace/api.py

```diff
@@ -226,7 +226,7 @@ def send_traces(self, traces):
                             payload.add_trace(trace)
                         except PayloadFull:
                             # If the trace does not fit in a payload on its own, that's bad. Drop it.
-                            log.warning('Trace %r is too big to fit in a payload, dropping it', trace)
+                            log.debug('Trace %r is too big to fit in a payload, dropping it', trace)
```

Comment:
We should keep this as a warning, but maybe we can format the message differently so it isn't as long.

Reply:
re: keeping as a warning. we really want to make sure these aren't hidden, we want people to know if a trace is every dropped so they can take actions. making a `debug` will effectively hide this from the customers